zeno_client/client.py

```diff
@@ -154,7 +155,7 @@ def upload_dataset(
         print("Successfully uploaded data")
 
     def upload_system(
-        self, name: str, df: pd.DataFrame, id_column: str, output_column: str
+        self, name: str, df: pd.DataFrame, *, id_column: str, output_column: str
```

Comment:
Should we swap these? 

And/or should we check that they are the right types in the body? AFAIK types are only suggestions, don't actually error if you pass the wrong type right?

Reply:
sorry mean swapping name and df to be consistent with upload dataset. IDK what is more natural.... maybe this is fine?

Reply:
No strong preference either, can swap and maybe make name named as well?

Hmm could check but we do nowhere else, right? Or which types would you check?

Reply:
yeah maybe swap and force named? I would just check that name is a string and that df is a dataframe. Maybe we should do checks across the board?

Reply:
I swapped them and forced named names as well now just df can be unnamed. 

Type-checking: could do but kinda weird for a python library, no? If we check we should probably do throughout and then in a separate PR.